src/transforms/babel.js

```diff
@@ -17,7 +17,8 @@ module.exports = async function(asset) {
     config.plugins = [require('@babel/plugin-transform-modules-commonjs')];
   }
 
-  let res = babel.transformFromAst(asset.ast, asset.contents, config);
+  let res = babel.transformFromAstSync(asset.ast, asset.contents, config);
```

Comment:
Maybe u could use the async one and convert it into a promise and use await babel.transformAst?
As the function this is called in is asynchronous anyway.

Reply:
Yes, do you have a preferred way for promisifing (like a helper) or can I create a Promise here?

Reply:
There is this, https://github.com/parcel-bundler/parcel/blob/master/src/utils/promisify.js